apps/meteor/app/mentions/server/server.js

```diff
@@ -28,7 +28,20 @@ export class MentionQueries {
 	}
 
 	getChannels(channels) {
-		return Rooms.find({ name: { $in: [...new Set(channels)] }, t: { $in: ['c', 'p'] } }, { fields: { _id: 1, name: 1 } }).fetch();
+		return Rooms.find(
+			{
+				$and: [
+					{
```

Comment:
This first `$and` is not doing anything.

Reply:
Thanks, didn't notice that 😞